docs/team/benfheng.adoc

```diff
@@ -1,68 +1,410 @@
+** Difficulties: Ensuring that all the information is easily accessible by the other features (MonthlyData, analytics and filter). Also ensuring that the application’s functionality is not affected by having optional inputs for transaction.
 
-* *Minor enhancement*: added a history command that allows the user to navigate to previous commands using up/down keys.
+* *Major enhancement*: Adding recurring transactions
+** What it does: It allows users add a transaction that is recurring for every subsequent month.  
+** Justification: This feature will allow users to set their monthly outgoing or incoming expenses such as insurance or their monthly income. It eases the need to constantly input the same transactions every month.
+** Highlights: This enhancement makes it more efficient and easier to use for users as they don’t have to constantly key in the transactions that recur every month. 
+** Depth of enhancement & completeness: Recurring transactions are all stored in the Global Data. Only when the user’s computer date reaches the new month, the new monthly recurring transactions are added to the transactions list. These changes are also automatically reflected in the monthly data panel. 
+** Difficulties: Ensuring that the recurring transactions will not prematurely affect the monthly data.
 
-* *Code contributed*: [https://github.com[Functional code]] [https://github.com[Test code]] _{give links to collated code files}_
+* *Major enhancement*: Import transactions from external file
+** What it does: It allows users to import their transactions from previously saved transaction lists.   
+** Justification: This feature will allow users to transfer their previous transactions from another device into different devices. 
+** Highlights: This enhancement makes it easier for users to save and reload their data anytime they require and view their expense statements.
+** Depth of enhancement & completeness: It works perfectly in tandem with the export function to help users to transfer their stored transactions to other devices, or just plain saving the information into csv files. 
+
+* *Minor enhancement*: Overhauled the UI of the application to make it more visually appealing to users.
+
+* *Code contributed*: [https://github.com/AY1920S2-CS2103-T09-3/main/commits/master?author=BenFheng[Commits]] [https://github.com/AY1920S2-CS2103-T09-3/main/pulls?q=is%3Apr+author%3ABenFheng[Pull Requests]][https://nus-cs2103-ay1920s2.github.io/tp-dashboard/#search=BenFheng&sort=groupTitle&sortWithin=title&since=2020-02-14&timeframe=commit&mergegroup=false&groupSelect=groupByRepos&breakdown=false&tabOpen=true&tabType=authorship&tabAuthor=HubertHalim&tabRepo=AY1920S2-CS2103-T09-3%2Fmain%5Bmaster%5D[RepoSense Code Contribution]]
 
 * *Other contributions*:
```

Comment:
I think this section is important and should not be omitted, just follow the template and link to PRs?